internal/pkg/plugin/common/helm/helm.go

```diff
@@ -23,6 +23,7 @@ type Options struct {
 	CreateNamespace bool `mapstructure:"create_namespace"`
 	Repo            helm.Repo
 	Chart           helm.Chart
+	DeploymentList  []string `mapstructure:"deploymentList"`
```

Comment:
Maybe nothing should be validate here.

Reply:
Because ArgoCD has the check item "DefaultDeploymentList", move the DeploymentList to the configuration yaml file for user configuration.
If not need to verify deployments,  will remove this item